src/View/Helper/PaginatorHelper.php

```diff
@@ -1110,6 +1155,8 @@ public function meta(array $options = [])
             $options['block'] = __FUNCTION__;
         }
         $this->_View->append($options['block'], $out);
+
+        return null;
```

Comment:
If you invert the logic, you will not have to do "return null;" at the end of a method.